src/amo/components/AddonReviewList.js

```diff
@@ -55,118 +106,101 @@ export class AddonReviewListBase extends React.Component {
     return `${this.addonURL()}reviews/`;
   }
 
-  renderReview(review: UserReviewType) {
+  renderReview({ review, key }: RenderReviewParams) {
     const { i18n } = this.props;
-    const timestamp = i18n.moment(review.created).fromNow();
+
+    let byLine;
```

Comment:
I feel like `byLine` is a bit of a niche word... I know that's what this is but `reviewAuthor` would work equally well for what this is and might be a bit easier to parse.